src/coro_rpc/benchmark/client/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ target_link_libraries(benchmark_client PRIVATE libcoro_rpc)
 target_include_directories(benchmark_client PRIVATE . ..)
 
 add_executable(benchmark_data_gen data_gen.cpp)
-target_link_libraries(benchmark_data_gen PRIVATE libcoro_rpc)
+target_link_libraries(benchmark_data_gen PRIVATE libcoro_rpc Threads::Threads)
```

Comment:
I have fixed the `-pthread` problem by https://github.com/alibaba/yalantinglibs/pull/30 yesterday.
please rebase to the latest main branch.